File: src/python_pkg/python_pkg/talker.py
from rclpy.node import Node
from std_msgs.msg import String


class Talker(Node):
    def __init__(self):
        super().__init__("talker")
        self.publisher = self.create_publisher(String, "python_chatter", 10)
        self.timer = self.create_timer(1.0, self.publish)

    def publish(self):
        message = String()
        message.data = "Help me Obi-Wan Kenobi, you're my only hope!"
        self.publisher.publish(message)

File: src/cpp_pkg/src/talker.cpp
#include <chrono>

#include "cpp_pkg/nodes.hpp"

Talker::Talker() : Node("talker") {
    publisher_ = create_publisher<std_msgs::msg::String>("cpp_chatter", 10);
    timer_ = create_wall_timer(std::chrono::seconds(1), [this] {
        std_msgs::msg::String message;
        message.data = "Help me Obi-Wan Kenobi, you're my only hope!";
        publisher_->publish(message);
    });
}
